sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -393,9 +405,15 @@ export enum ErrorNameConditionMapper {
     UnauthorizedError = "amqp:unauthorized-access"
 }
 
+// @public
+export function isCredential(thing: unknown): thing is TokenCredential | NamedKeyCredential | SASCredential;
```

Comment:
Nit:

I am leaning towards either moving the `isCredential()` definition out of core-amqp and into EH+SB or have the checks inlined in EH + SB i.e. `if (isTokenCredential(thing) || isNamedKeyCredential(thing) || isSASCredential(thing))` instead of `if isCredential(thing)`.

This is to keep the option open for having yet another credential that might be specific to EH or SB

I see core-amqp as having the building blocks that are then used by EH and SB

Thoughts?

Reply:
Agreed. It keeps the added APIs to core-amqp small.